Explorer/Assets/DCL/Interaction/Systems/ProcessPointerEventsSystem.cs

```diff
@@ -167,6 +163,15 @@ private void ResetHighlightComponent(ref HighlightComponent highlightComponent)
                 // Try Append Hover Feedback
                 HoverFeedbackUtils.TryAppendHoverFeedback(sdkInputActionsMap, pointerEvent,
                     ref hoverFeedbackComponent, anyInputInfo.AnyButtonIsPressed);
+
+            }
+
+            if (!isAtDistance) return false; //This only works if elements have PBPointerEvents to filter distances, otherwise, we need to send the input event anyway (not done yet)
```

Comment:
What happens if the last pointer event iterated is at distance, but the rest is not? Or the other way around, all events not at distance but the last it is. Doesn't feel consistent to me

Reply:
its the logic we had already tbh, I dont like it much either, but the PointerEvent component only has information about some events not all of them, and if the event we are looking for is not defined in the list then we must assume it respects the distance of one of the previous ones (otherwise for example with the billboards in the top of the plaza, you can interact from any distance, as the F and E behaviours are NOT defined in the PointerEvent component)